src/khoj/interface/web/chat.html

```diff
@@ -207,7 +218,8 @@
 
         <!-- Chat Footer -->
         <div id="chat-footer">
-            <input type="text" id="chat-input" class="option" onkeyup=incrementalChat(event) autofocus="autofocus" placeholder="What is the meaning of life?">
+<!--            <input type="text" id="chat-input" class="option" onkeyup=incrementalChat(event) autofocus="autofocus" placeholder="What is the meaning of life?">-->
```

Comment:
Can we remove commented out code?

Reply:
Yes! `input` box is no longer used.

Reply:
If it's not required, can you please create another commit to remove it?